azure-pipelines.yml

```
@@ -0,0 +1,34 @@
+# ASP.NET Core (.NET Framework)
+# Build and test ASP.NET Core projects targeting the full .NET Framework.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- master
```

I think this trigger should point to the branch

---

Agreed you probably want to pick up all assignment branches
```suggestion
- Assignment*
```